### PR TITLE
Certificate -> CertificateRequest on rpc.proto

### PR DESCRIPTION
**Pull request by andresilva91**
## Motivation

The current proto struct is called `Certificate`, but it isn't really representing that

## Proposal

Rename it to what it actually represents

## Test Plan

CI

### CODE REVIEW

**Comment by andresilva91** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/2084?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2084** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/2084?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2081** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/2081?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2079** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/2079?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2074** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/2074?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @andresilva91 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: linera-rpc/proto/rpc.proto [hunk 1]
```diff
@@ -197,19 +197,10 @@ message LiteCertificate {
 
 // A certified statement from the committee, together with other certificates
 // required for execution.
-message Certificate {
+message CertificateRequest {
```
Comment (afck) [inline linera-rpc/proto/rpc.proto:200]: Maybe `HandleCertificateRequest`? We're not requesting a certificate, after all.

File: linera-rpc/proto/rpc.proto [hunk 2]
```diff
@@ -197,19 +197,10 @@ message LiteCertificate {
...
-  bytes value = 2;
-
-  // The round in which the value was certified.
-  bytes round = 3;
-
-  // Signatures on the value hash and round
-  bytes signatures = 4;
-
   // Other certificates containing bytecode required by the first one
   bytes hashed_certificate_values = 5;
```
Comment (MathieuDutSik) [inline linera-rpc/proto/rpc.proto:205]: Why not renumber the entries of the proto file?

File: linera-rpc/src/grpc/server.rs [hunk 3]
```diff
@@ -510,7 +509,7 @@ where
     #[instrument(target = "grpc_server", skip_all, err, fields(nickname = self.state.nickname(), chain_id = ?request.get_ref().chain_id()))]
     async fn handle_certificate(
         &self,
-        request: Request<Certificate>,
+        request: Request<api::CertificateRequest>,
```
Comment (MathieuDutSik) [inline linera-rpc/src/grpc/server.rs:512]: Maybe, we should have a `use api::CertificateRequest` on top and here replace `api::CertificateRequest` by `CertificateRequest`. If only for consistency with the rest of the code.

Comment (andresilva91) [reply linera-rpc/src/grpc/server.rs:514]: After the rename of `HandleCertificateRequest` to `CertificateRequest`, I can't

File: linera-rpc/src/grpc/client.rs [hunk 4]
```diff
@@ -171,7 +171,7 @@ impl ValidatorNode for GrpcClient {
         delivery: CrossChainMessageDelivery,
     ) -> Result<linera_core::data_types::ChainInfoResponse, NodeError> {
         let wait_for_outgoing_messages = delivery.wait_for_outgoing_messages();
-        let request = HandleCertificateRequest {
```
Comment (ma2bd) [inline linera-rpc/src/grpc/client.rs:174]: Sorry but `HandleCertificateRequest` was correct

**Review by afck — APPROVED** [review]

**Comment by andresilva91** [comment]
### Merge activity

* **Jun 18, 8:14 AM EDT**: @andresilva91 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/linera-io/linera-protocol/2084).

**Pull request merged**